dlpy/layers.py

```diff
@@ -2097,6 +2097,9 @@ class Reshape(Layer):
         Specifies the depth of the feature maps.
     src_layers : iter-of-Layers, optional
         Specifies the layers directed to this layer.
+    order : string, optional
```

Comment:
can we please also explain one or two of these options. something like whc is width x height x number of channels (or depth), auto means this and so on.

Reply:
Added more descriptions.